### DISCUSSION TITLE
Description should have an image preview

### DISCUSSION THREAD

**Discussion by andupotorac** [AlUlkesh/stable-diffusion-webui-images-browser#206] [General]
I find it very odd this is an extension that will alter our UI, yet we don't see how it should look like before we even install it...

**Comment by AlUlkesh** [comment]
What gives you the impression, that it will alter the UI?

It doesn't, it adds a new tab.

**Reply by andupotorac** [reply]
Right, but shouldn't we see how it will look like before we install it?

**Reply by AlUlkesh** [reply]
Makes sense. I'm currently working on updating the UI, after that I can put some examples in the readme.

**Reply by andupotorac** [reply]
> Makes sense. I'm currently working on updating the UI, after that I can put some examples in the readme.

Thanks!
